docs/docs/integrations/vectorstores/supabase.ipynb

```diff
@@ -197,7 +197,7 @@
    "id": "5abb9b93",
    "metadata": {},
    "source": [
-    "Insert the above documents into the database. Embeddings will automatically be generated for each document."
+    "Insert the above documents into the database. Embeddings will automatically be generated for each document. You can adjust the chunk_size based on the amount of documents you have. The default is 500 but lowering it may be necessary"
```

Comment:
```suggestion
    "Insert the above documents into the database. Embeddings will automatically be generated for each document. You can adjust the chunk_size based on the amount of documents you have. The default is 500 but lowering it may be necessary."
```